### ISSUE TITLE
syn49637038 - BloodAge10xArtyomov

### ISSUE THREAD

**Issue by arschat** [ebi-ait/hca-ebi-wrangler-central#1222]
### Project short name:
BloodAge10xArtyomov

### Primary Wrangler:
Marina Terekhova / Arsenios

### Associated files

* Google Drive: [folder](https://drive.google.com/drive/folders/1cvPS3dsqZqFD1XD13yKCLZrWm5BZHuQg?usp=sharing), [spreadsheet](https://docs.google.com/spreadsheets/d/1T_CaGacnz3SXO9EYH4ph7OtX0OOVcwyP/edit?usp=drive_web&ouid=101176945353039944995&rtpof=true)

### Published study links

* Paper: [Single-cell atlas of healthy human blood unveils age-related loss of NKG2C+GZMB-CD8+ memory T cells and accumulation of type 2 memory T cells](https://doi.org/10.1016/j.immuni.2023.10.013)

* Accessioned data:
synapse: [syn49637038](https://www.synapse.org/#!Synapse:syn49637038/files/)
* ingest uuid: [896f377c-8e88-463e-82b0-b2a5409d6fe4](https://contribute.data.humancellatlas.org/projects/detail?uuid=896f377c-8e88-463e-82b0-b2a5409d6fe4)
* hca-util upload area `cffd58a6-a629-4fa0-8751-c695b8657bd2`

### Key Events

- [ ] Convert published metadata to HCA spreadsheet
- [ ] Manually curate dataset to meet HCA metadata standard
- [ ] Collect any matrix and cell-type annotation files
- [ ] Are the analysis files suitable for CellxGene? If something is missing get in touch with the authors to request it
- [ ] Upload sheet to validate metadata
- [ ] Transfer raw files to ingest to validate data files
- [ ] Check linking using ingest graph validator
- [ ] Ask the Secondary Wrangler for an end-to-end review of the project. Ask the Expertise Wrangler to review specific tabs if needed
- [ ] Submit dataset to Production 
- [ ] Complete the Export SOP
- [ ] Convert project data to SCEA format following the SCEA conversion SOP if appropriate

**Comment by arschat** [comment]
After [this](https://groups.google.com/a/data.humancellatlas.org/g/wrangler-team/c/O7nv0MSe8TE/m/TZ3KT1VSAAAJ) communications, contributor has submitted this spreadsheet.
[hca_template_Artyomov_MT.xlsx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM3lDQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--61a075637d1758a7341b4aaba28a7aa9383f565f/hca_template_Artyomov_MT.xlsx)

**Comment by arschat** [comment]
## Notes
- There are multiple specimens from the same donor in different time points and ages as well as BMIs. This cannot be recorded in our fields for now, but a workaround is to capture that in the description field at the specimen level.
- The machine used for sequencing (Illumina NovaSeq S4) is not available in the current [OLS](https://ontology.archive.data.humancellatlas.org/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0002699)
- Processes in seq_files have yet to be grouped together

**Comment by arschat** [comment]
## Notes
- There are multiple specimens from the same donor in different time points and ages as well as BMIs. This cannot be recorded in our fields for now, but a workaround is to capture that in the description field at the specimen level.
- The machine used for sequencing (Illumina NovaSeq S4) is not available in the current [OLS](https://ontology.archive.data.humancellatlas.org/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0002699)
- Processes in seq_files have yet to be grouped together

**Comment by arschat** [comment]
There is no machine named `Illumina NovaSeq S4`, but S4 can be the type of the cell flow used. 
Based on the [illumina website](https://emea.illumina.com/systems/sequencing-platforms.html), there are two NovaSeq series available, Illumina NovaSeq 6000 and Illumina NovaSeq X. From those two, only the Illumina NovaSeq 6000 has an S4 cell flow type option. Therefore, this is the sequencer that is used.

**Comment by arschat** [comment]
@anu-shiva to sec review

**Comment by anu-shiva** [comment]
Looks great- very tricky dataset!

**Project contributors**
- Most of the authors are affiliated with another department- if you would like to include this information.

**Donor**
- Could there be a way to specify the BMI range (like the age range is allowed) or at least a mention that the BMI value is for the first visit? I see that the precise BMIs are described in the specimen description, however, the donor BMIs can be a bit confusing with a single BMI value while a range of ages is recorded.

**Specimen**
- Could this be added to the time course - Each age group followed over the course of 3 years with annual blood collection.


**Collection protocol**
- 500g to be replaced with 500 × g.
- Enrichment is duplicated here and can be removed as it is included in the enrichment tab

**Library prep**
- The library prep descriptions look swapped- FB has TCR, TCR has BCR, and BCR has FB.
- cDNA amplification method can be added.
- primer can be added
- The UMI offset is 16 and the UMI length is 10 for 10x5v2 in the assay cheat sheet

**Sequencing protocol**
- The current description applies to the gene expression libraries. The sequencing info for others (Enriched and Feature Barcode libraries were sequenced on a NovaSeq S4 (300 cycles) flow cell, targeting 5,000 read pairs per cell) can be added too in the same or different entry.

**Analysis protocol**
- I think there is some description in the paper on the normalization steps- as the final matrices provided are normalized counts. (don't see RPM, TPM though). The raw matrix generation could possibly be replaced by normalized if we have enough necessary info?

**Analysis files**
- four files are available as _.tgz_ extension- unless you changed all files to .gz.

**Comment by arschat** [comment]
Thanks for the review Anu!
I incorporated all your comments, I have a few notes here.

> Donor
Could there be a way to specify the BMI range (like the age range is allowed) or at least a mention that the BMI value is for the first visit? I see that the precise BMIs are described in the specimen description, however, the donor BMIs can be a bit confusing with a single BMI value while a range of ages is recorded.

About that, ideally I would like to have BMI in range too, and specify in the specimen description the exact BMI in the time of collection. However, BMI does not allow an hyphen in the regex of the [schema](https://schema.humancellatlas.org/module/biomaterial/1.0.9/human_specific) therefore we cannot add range there. Since that's what the contributor filled I thought that this would be the best way to model it.

> Specimen
Could this be added to the time course - Each age group followed over the course of 3 years with annual blood collection.

Thanks for the tip, I added the timecourse on `time after first visit` for 2nd and 3rd visit's specimens based on the Visit_2/3_Age - Visit_1_Age formula

> Analysis files
four files are available as .tgz extension- unless you changed all files to .gz.

The contributor filled the spreadsheet for all the analysis files as .tar.gz instead of 4 files as .tar.tgz . I made sure all files were compressed with the same .tar.gz compression and named them likewise before uploading them.

- [ ] update submission
- [ ] graph valid
- [ ] export
- [ ] import form sent
